src/server/robots.js

```diff
@@ -1,6 +1,14 @@
+const fs = require('fs')
+const themePath = require('../../core/build/theme-path')
+const robots = `${themePath}/robots.txt`
 
-module.exports = (expressApp) => {
+module.exports = expressApp => {
   expressApp.get('/robots.txt', (req, res) => {
-    res.end('User-agent: *\nDisallow: ')
+    if (fs.existsSync(robots)) {
+      res.type('text/plain')
+      res.sendFile(robots)
+    } else {
+      res.end('User-agent: *\nDisallow: ')
```

Comment:
I believe it might be even a simpler solution just adding a static file server for robots.txt in the `core/scripts/server.js` (https://github.com/DivanteLtd/vue-storefront/blob/e14a2a0abde6f71a48e21c8e694b5502e50d3aa3/core/scripts/server.js#L112) + creating a default file with the `User-agent: *\nDisallow`)

Reply:
@pkarw don't you think such case-specific things shouldn't land in core?

Reply:
Maybe we should have a directory in theme that maps directly to server root path so people can put there whatever they want (we should also place assets in there)

Reply:
Sure, there should be assets server bound to robots.txt from these exactly as we do have with the assets  (see example above)